crowdsourcing/serializers/project.py

```diff
@@ -36,12 +36,15 @@ class ModuleSerializer(DynamicFieldsModelSerializer):
     deleted = serializers.BooleanField(read_only=True)
     template = TemplateSerializer(many=True, read_only=False)
     module_tasks = TaskSerializer(many=True, read_only=True)
+    csv_data = serializers.CharField(read_only=True)
 
     def create(self, **kwargs):
         templates = self.validated_data.pop('template')
         project = self.validated_data.pop('project')
+        csv_data = self.initial_data.pop('csv_data')
```

Comment:
I'm not sure whether you have validate the `csv_data` because you get it right away from `initial_data`. CMIIW.